kubernetes/linux/Dockerfile

```diff
@@ -15,6 +15,7 @@ ENV HOST_VAR /hostfs/var
 ENV AZMON_COLLECT_ENV False
 ENV KUBE_CLIENT_BACKOFF_BASE 1
 ENV KUBE_CLIENT_BACKOFF_DURATION 0
+ENV RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR 0.9

```

Comment:
how much mem improvements you see here due to this, and also whats the cost of cpu due to frequent gc (especially on smaller clusters) ?

Reply:
Around ~100MB in large clusters and verified that no significant impact  on the CPU on small cluster (26 nodes and 890 pods).  Here is the cpu usage of the  cluster with RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR = 0.9 and without this.
![image](https://user-images.githubusercontent.com/20712167/102137900-68962000-3e10-11eb-8cdc-db2a8bd7ca52.png)